test/aws-s3/aws-s3-to-http.yaml

```
@@ -15,8 +15,8 @@
 # limitations under the License.
 # ---------------------------------------------------------------------------
 
-apiVersion: camel.apache.org/v1alpha1
-kind: KameletBinding
+apiVersion: camel.apache.org/v1
+kind: Pipe
 metadata:
   name: aws-s3-to-http
 spec:
```

```suggestion
spec:
  integration:
    dependencies:
    - "camel:cloudevents"
```

This is the only YAKS test failing. We need to add "camel-cloudevents" dependency because the data type transformer is now provided by Camel 4.0.1 and uses this as provided dependency.

Hopefully this fixes the test @apupier @oscerd